packages/gmx/src/test-setup.ts

```diff
@@ -105,3 +107,53 @@ export const failingTestCasesV2 = [
   }),
   createTestCase(REDUCE_ORDER_V2, 'when wrong OrderType is used'),
 ]
+
+export const passingOptionsTestCases = [
+  createTestCase(MARKET_SHORT_ETH_ORDER_V2, 'when using a market order'),
+  createTestCase(LIMIT_LONG_ARB_ORDER_V2, 'when using a limit order'),
```

Comment:
would also suggest running some tests with some or all of the parameters undefined ie: any token quests

check line 74 for example